ThesisTemplate.tex

```diff
@@ -381,29 +377,29 @@ \section{Nuclear Fuel Cycle Simulators}
 
 The nuclear fuel cycle shares similar enough underlying characteristics with a nuclear hybrid energy system to loosely model the system.  A more thorough analysis of the benefits and challenges to using a NFCS to model NRHES will be discussed surrounding the model. A NFCS that is in a mature state of development, open source, easy to incorporate new components, flexible enough to change the materials exchanged rapidly, and able to be coupled with external tools for modeling the renewable resource could speed up the development of a NRHES model. Cyclus is the only currently open source nuclear fuel cycle simulator.  As such, Cyclus has distinct advantages among the NFCSs when developing an NRHES.
 
 \section{Cyclus}
 
-The flexibility and modular design of Cyclus suggest its potential to successfully model a NRHES. The Cyclus fuel cycle simulator's underlying system is the dynamic resource exchange (DRE) \cite{Huff2016}.  The DRE has a set time step at which point certain facilities will put their resources on the market for exchange while other facilities can make a bid for the desired amount of the resource. For example, in the case of modeling the nuclear fuel cycle, the reactor could declare how much spent fuel it has once every eighteen months.  A reprocessing facility and a long-term geologic repository could both bid on how much of the spent fuel each facility could take at that point.  The DRE has a built in functionality of preferences for allocating a resource.  For the example given, a preference could be a built in for the reprocessing facility, meeting the bid from this facility before sending the remaining spent fuel resource to the long-term geologic repository. The simulation could be run over hundreds of years to demonstrate the accumulation of spent fuel and fission products over the entire timeline. 
+The flexibility and modular design of Cyclus suggest its potential to successfully model a NRHES. The Cyclus fuel cycle simulator's underlying system is the dynamic resource exchange (DRE) \cite{Huff2016}.  The DRE has a set time step at which point certain facilities will put their resources on the market for exchange while other facilities can make a bid for the desired amount of the resource. For example, in the case of modeling the nuclear fuel cycle, the reactor could declare how much spent fuel it has once every eighteen months.  A reprocessing facility and a long-term geologic repository could both bid on how much of the spent fuel each facility could take at that point.  The DRE has a built in functionality of preferences for allocating a resource.  For the example given, a preference could be a built in for the reprocessing facility, meeting the bid from this facility before sending the remaining spent fuel resource to the long-term geologic repository. The simulation could be run over hundreds of years to demonstrate the accumulation of spent fuel and fission products over the entire timeline.
```

Comment:
Are you going to explain Cyclus in more detail later in the thesis? You want to demonstrate your expertise.

Reply:
I will explain Cyclus later.